pysatNASA/instruments/methods/iss.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Full license can be found in License.md
+# Full author list can be found in .zenodo.json file
+# DOI:10.5281/zenodo.3986131
+#
+# DISTRIBUTION STATEMENT A: Approved for public release. Distribution is
+# unlimited.
+# ----------------------------------------------------------------------------
+"""Provides non-instrument specific routines for ISS data."""
```

Comment:
```suggestion
"""Provides non-instrument specific routines for International Space Station (ISS) data."""
```

Reply:
We haven't spelled out acronyms elsewhere in methods, also is too many characters. Add to agenda.